ports/librtmp/vcpkg.json

```
@@ -1,7 +1,6 @@
 {
   "name": "librtmp",
-  "version-date": "2019-11-11",
-  "port-version": 4,
+  "version": "2.6",
   "description": "RTMPDump Real-Time Messaging Protocol API",
   "homepage": "https://rtmpdump.mplayerhq.hu",
   "dependencies": [
```

License `librtmp license: LGPLv2.1` from https://git.ffmpeg.org/gitweb/rtmpdump.git/blob/HEAD:/README#l7
```suggestion
  "dependencies": [
  "license": "LGPL-2.1",
```

---

I added the license.